rom/dev/src/start_fpga.S

```diff
@@ -0,0 +1,354 @@
+/*++
```

Comment:
This is too much copy-and paste for my liking.

I think we should use the preprocessor, like in https://github.com/chipsalliance/caliptra-sw/pull/428